src/app/application/components/app-collection/AppCollection.vue

```diff
@@ -175,7 +175,7 @@ const lastPageNumber = ref(props.pageNumber)
 const lastPageSize = ref(props.pageSize)
 
 const columnWidth = computed(() => {
-  const specialColumns = props.headers.filter((header) => ['warnings', 'actions'].includes(header.key))
+  const specialColumns = props.headers.filter((header) => ['details', 'warnings', 'actions'].includes(header.key))
```

Comment:
I think you mentioned this elsewhere but I also would like to avoid these kinda of special things in a generic component.

Reply:
It does have its caveats, but it’s better to standardize this in one place than to rely on all users of AppCollection to apply the exact same column sizing styles.

We also wanted to have the behavior that columns, by default, are spread out equally. Together with the desirable behavior of the special columns having a smaller, mostly fixed width, we need to specify columns widths for all columns which brings the need for this kind of logic.